docker-compose.test.yml

```diff
@@ -43,24 +43,6 @@ services:
       - CCLF_IMPORT_STATUS_RECORDS_INTERVAL=10
       - BCDA_SSAS_CLIENT_ID=fake-client-id
       - BCDA_SSAS_SECRET=fake-secret
-      - SSAS_ADMIN_SIGNING_KEY_PATH=../../../shared_files/ssas/admin_test_signing_key.pem
-      - SSAS_PUBLIC_SIGNING_KEY_PATH=../../../shared_files/ssas/public_test_signing_key.pem
-      - SSAS_PUBLIC_PORT=:3003
-      - SSAS_ADMIN_PORT=:3004
-      - SSAS_HTTP_TO_HTTPS_PORT=:3005
-      - SSAS_READ_TIMEOUT=10
-      - SSAS_WRITE_TIMEOUT=20
-      - SSAS_IDLE_TIMEOUT=120
-      - SSAS_HASH_ITERATIONS=130000
-      - SSAS_HASH_KEY_LENGTH=64
-      - SSAS_HASH_SALT_SIZE=32
-      - SSAS_DEFAULT_SYSTEM_SCOPE=bcda-api
-      - SSAS_MFA_CHALLENGE_REQUEST_MILLISECONDS=0
-      - SSAS_MFA_TOKEN_TIMEOUT_MINUTES=60
-      - SSAS_MFA_PROVIDER=${SSAS_MFA_PROVIDER}
-      - SSAS_TOKEN_BLACKLIST_CACHE_CLEANUP_MINUTES=15
-      - SSAS_TOKEN_BLACKLIST_CACHE_TIMEOUT_MINUTES=1440
-      - SSAS_TOKEN_BLACKLIST_CACHE_REFRESH_MINUTES=5
```

Comment:
Do we need a `bcda-ops` PR in addition to this PR to clean up any extraneous env vars?

Reply:
Thankfully, no--these env vars are only set for SSAS.